sycl/test-e2e/Adapters/enqueue-arg-order-image.cpp

```diff
@@ -1,5 +1,5 @@
 // REQUIRES: aspect-ext_intel_legacy_image
-// UNSUPPORTED: target-amd
+// UNSUPPORTED: target-amd || target-native_cpu
```

Comment:
If the comment below doesn't apply to `native_cpu`, it should have its own `UNSUPPORTED` line and explanation.